tests/setup.js

```diff
@@ -3,6 +3,9 @@ import config from 'config';
 import areIntlLocalesSupported from 'intl-locales-supported';
 import values from 'object.values';
 
+import 'jest-enzyme';
```

Comment:
Actually, just checking now it seems that was only required for earlier versions of Jest. I removed that line and all the tests still work/pass. I haven't used much/any of the `jest-enzyme` integration yet though, if it's needed for that I'll keep it in but add a comment to explain.